statistics/old_update(DO_NOT_RUN).sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 # Expected filename for addFields.js
+#DO NOT RUN THIS FILE UNLESS YOU KNOW WHAT YOU'RE DOING
+#Uncomment the lines below for full functionality
```

Comment:
For safety, remove this files if it doesn't work anymore. We can always bring it back from the git history.
I think this applies to most comments about "old" ways of doing things :)

Reply:
New to git, seems reasonable!